completion_aggregator/api/v1/views.py

```diff
@@ -541,7 +548,10 @@ def get(self, request, course_key):
         """
         Handler for GET requests
         """
-        course_key = CourseKey.from_string(course_key)
+        try:
+            course_key = CourseKey.from_string(course_key)
+        except InvalidKeyError:
+            raise NotFound("Invalid block key: '{}'.".format(course_key))
```

Comment:
This one is a "course key" too.

Reply:
sure, I'll take care of those first thing Monoday